### PR TITLE
Feature/fsa 856 table formatting

### PR DESCRIPTION
**Pull request by Roxana-deeson**
- added `id` attribute to th of mixed tables (mixed = tables that have row and column headers) and `heading` attribute to td in these tables. This way we see clearly which td relates to what th
-  design of mixed tables is slightly different for better readability
-  tables that have > 3 columns don't stack anymore on smaller screen, it scrolls horizontally instead

### CODE REVIEW

File: src/lib.scss [hunk 1]
```diff
@@ -793,6 +793,12 @@ $mobileGutter: 1rem;
     }
   }
 
+  &.fsa-table--mixed tbody th {
+    background: none;
```
Comment (c-ancia) [inline src/lib.scss:797]: do we need the background none explicitly? 🤔

File: src/components/article/TextBlock/textBlock.js [hunk 2]
```diff
@@ -51,40 +38,52 @@ export default function () {
...
+            allTh.forEach((heading) => {
+              const identifier = heading.innerHTML.toLocaleLowerCase();
+              heading.setAttribute('id', identifier);
+            });
+            table.rows.forEach((tr) => {
+              const th = tr.cells[0].getAttribute('id');
+              const td = tr.querySelectorAll('td');
+              td.forEach((cell) => {
+                const headerCell = table.rows[0].cells[cell.cellIndex];
+                const headerLabel = headerCell.getAttribute('id') + ' ' + th;
```
Comment (c-ancia) [inline src/components/article/TextBlock/textBlock.js:66]: isn't the th taken as part of `allTh`?
Also, what happens if the row doesn't have a th and it's only column ths? (hope this makes sense ahah)

**Review by c-ancia — CHANGES_REQUESTED** [review]
there are loads of changes in the changelog (like added lines etc) can you check please?

File: src/lib.scss (hunk 1 above)
Comment (Roxana-deeson) [reply src/lib.scss:797]: yes because it's purple otherwise :D or should i do background: white instead? 🤔

**Review by c-ancia — APPROVED** [review]

**Pull request merged**